azext_edge/tests/edge/init/int/test_init_int.py

```diff
@@ -150,3 +344,11 @@ def _process_additional_args(additional_args: str) -> Dict[str, Union[str, bool]
         else:
             arg_dict[arg[0]] = arg[1]
     return arg_dict
+
+
+def _strip_quotes(argument: Optional[str]) -> Optional[str]:
+    if not argument:
+        return argument
+    if argument[0] == argument[-1] and argument[0] in ("'", '"'):
+        argument = argument[1:-1]
```

Comment:
this could be `argument = argument.strip("\"'")`

Reply:
I want to make sure that I don't remove an extra " or ' in the case of:
`'az iot ops create {blah} --description "my awesome cluster"'`